### DISCUSSION TITLE
Discord Support Server

### DISCUSSION THREAD

**Discussion by infinitytmbots** [Ayfri/Advanced-Command-Handler#73] [Q&A]
Hello there. First of all, I would like to thank you for such an amazing command handler whose fork powers our bot in 2.5K servers. 

I couldn't find a listing anywhere, so I would like to enquire if there is an official discord server relating to this command handler.

**Comment by Ayfri** [comment]
Actually, there is not, you can maybe join my personal Discord Server, [here is the invite](https://discord.gg/BySjRNQ9Je).
You can also directly talk to me on Discord, I am `Ayfri#0453`.

If a lot of people needs a server, I will create one ^^
